src/components/Header/header.module.css

```diff
@@ -0,0 +1,158 @@
+.header {
+  background-color: var(--darkest-grey);
+  padding: 0 30px;
+}
+
+.headerContainer {
+  max-width: 1280px;
+  padding: 15px 0;
+  margin: 0 auto;
+  display: flex;
+  align-items: center;
+}
+
+.headerLogoIcon {
+  fill: var(--white);
+}
+
+.headerThemeModeBtns {
+  display: none;
+}
+
+.headerThemeModeBtn,
+.headerCartBtn,
+.headerBurgerBtn,
+.headerAuthBtn {
+  cursor: pointer;
+  background-color: transparent;
+  border: none;
+}
+
+.headerThemeModeBtn {
+  padding: 10px 5px;
+}
+
+.headerThemeModeBtnIcon {
+  fill: none;
+  stroke: var(--grey-2);
+  transition: .3s stroke ease-in-out;
+}
+
+.headerThemeModeBtn.headerThemeModeBtnActive {
+  pointer-events: none;
+}
+
+.headerThemeModeBtn.headerThemeModeBtnActive .headerThemeModeBtnIcon {
+  stroke: var(--white);
+}
+
+.headerCartBtn {
+  margin-left: auto;
+}
+
+.headerCartIcon {
+  fill: none;
+}
+
+.headerCartIcon path {
+  stroke: var(--white);
+  stroke-width: 1.5px;
+}
+
+.headerBurgerBtn {
+  margin-left: 14px;
+}
+
+.headerPageLinks {
+  display: none;
+}
+
+.headerPageLink {
+  text-decoration: none;
+  color: var(--white);
+  font-weight: 500;
+  transition: .3s font-weight ease-in-out;
```

Comment:
```suggestion
    min-width: 80px; // we could play with this number to fit the desgin
    display: flex;
    justify-content: center;
```

Reply:
my bad, fixed)

Reply:
I was thinking about this solution too) But how to implement it to the footerlink?..